ert_logging/logger.conf

```diff
@@ -5,9 +5,9 @@ formatters:
 handlers:
   console:
     class: logging.StreamHandler
-    level: WARNING
+    level: INFO
     formatter: simple
-    stream: ext://sys.stdout
+    stream: ext://sys.stderr
```

Comment:
i'm not sure i agree on this - putting INFO msgs in stderr? 🤔

Reply:
Yep. All logs should go to `stderr`. `stdout` is only for output that you actually want to use. From `man stderr`:

> Under  normal circumstances every UNIX program has three streams opened
> for it when it starts up, one for input, one for output,  and  one  for
> **printing diagnostic or error messages**.  These are typically attached to
> the user's terminal (see tty(4) but might instead  refer  to  files  or
> other  devices,  depending  on what the parent process chose to set up.
> (See also the "Redirection" section of sh(1).)

Logs are diagnostic messages, not output.